entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+source /includes/colors.sh
+
+
+if [[ "${PUID}" -eq 0 ]] || [[ "${PGID}" -eq 0 ]]; then
+    pp --warning "[WARNING] Running as root is not supported, please fix your PUID and PGID!\n"
+    exit 1
+elif [[ "$(id -u steam)" -ne "${PUID}" ]] || [[ "$(id -g steam)" -ne "${PGID}" ]]; then
+    pp --info "Old steam user UID was '$(id -u steam)' and GID was '$(id -g steam)'\n"
+    pp --info "Setting steam user UID to '${PUID}' and GID to '${PGID}'\n"
+    usermod -o -u "${PUID}" steam
+    groupmod -o -g "${PGID}" steam
```

Comment:
i like that approach. my workaround was similar but thats even better.

Reply:
Using another popular repos to see how people are handling their problems and seeing how they approach it, but always with a foot back. I tested it, checked for security issues it might create, and only then I used this approach.
Wouldn't want to introduce a vulnerability in this beautiful project